frontend/javascripts/viewer/view/ai_jobs/alignment/ai_alignment_settings.tsx

```
@@ -1,11 +1,23 @@
-import { Card, Col, Collapse, ConfigProvider, Form, Input, Row, Space } from "antd";
+import {
+  Card,
+  Checkbox,
+  Col,
+  Collapse,
+  ConfigProvider,
+  Form,
+  Input,
+  Row,
+  Space,
+  Tooltip,
```

Biome (2.5.7)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)